packages/storybook/stories/va-icon-uswds.stories.tsx

```diff
@@ -16,7 +16,7 @@ export default {
 
 const defaultArgs = {
   icon: 'alarm',
-  size: 4,
+  size: 3,
```

Comment:
Just double checking about this change. Did you intentionally want to make the icons smaller?